doc/rst/technotes/gpu.rst

```diff
@@ -140,17 +140,16 @@ before building Chapel.
 
 Chapel's build system will automatically try and deduce what type of GPU you
 have and where your installation of relevant runtime (e.g. CUDA or ROCM) are.
-If the type of GPU is not detected you may set ``CHPL_GPU_CODEGEN`` manually to
-either ``cuda`` (for NVIDIA GPUs) or ``rocm`` (for AMD GPUs). If the relevant
-runtime path is not automatically detected (or you would like to use a
-different installation) you may set ``CHPL_CUDA_PATH`` and/or
-``CHPL_ROCM_PATH``.
+If the type of GPU is not detected you may set ``CHPL_GPU`` manually to either
+``nvidia`` or ``amd`` . If the relevant runtime path is not automatically
+detected (or you would like to use a different installation) you may set
+``CHPL_CUDA_PATH`` and/or ``CHPL_ROCM_PATH``.
 
 ``CHPL_GPU_ARCH`` environment variable can be set to control the desired GPU
 architecture to compile for.  The default value is ``sm_60`` for
-``CHPL_GPU_CODEGEN=cuda`` and ``gfx906`` for ``CHPL_GPU_CODEGEN=rocm``. You may
-also use the ``--gpu-arch`` compiler flag to set GPU architecture. For a list
-of possible values please refer to `CUDA Programming Guide
+``CHPL_GPU=nvidia`` and ``gfx906`` for ``CHPL_GPU=amd``. You may also use the
```

Comment:
I did some more grepping and fixed some stuff.

Note that this PR kinda asks for yet another rename in our nightly tests. But my stance is that we don't touch them just yet to avoid churn. We can revisit when:

- either we make `array_on_device` the default, which will require some shuffling, anyways, or
- when we add intel support.

Reply:
Having gone through it once, it's not that difficult to rename the jobs (just somewhat nervewracking). But yeah I agree I'm inclined to let the names sit untouched unless we need to rename them anyway or someone speaks up that they're sufficiently irked by the inconsistencies.